content/mapping/product/0.0.13/schema

```diff
@@ -0,0 +1,286 @@
+      "additionalProperties": { "$ref": "#/$defs/property" }
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "property": {
+      "type": "object",
+      "properties": {
+        "title": { "$ref": "#/$defs/title" },
+        "description": { "$ref": "#/$defs/description" },
+        "type": {
+          "enum": [
+            "string",
+            "number",
+            "boolean",
+            "array"
+          ]
+        },
+        "format": {
+          "enum": [
+            "date-time",
+            "uri"
+          ]
+        },
+        "items": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "enum": [
+                "string"
+              ]
+            },
+            "enum": {
+              "type": "array"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["type"],
+          "dependentSchemas": {
+            "enum": {
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "string"
+                  }
+                }
+              },
+              "then": {
+                "properties": {
+                  "enum": {
+                    "items": {
+                      "type": "string"
+                    }
+                  }
+                }
+              },
+              "required": ["type"]
+            }
+          }
+        },
+        "minLength": {
+          "type": "integer",
+          "minimum": 0
+        },
+        "maxLength": {
+          "type": "integer",
+          "minimum": 0
+        },
+        "pattern": {
+          "type": "string",
+          "format": "regex"
+        },
+        "minimum": {
+          "type": "number"
+        },
+        "maximum": {
+          "type": "number"
+        },
+        "enum": {
+          "type": "array"
+        }
+      },
+      "additionalProperties": false,
+      "if": {
+        "properties": {
+          "type": {
+            "const": "array"
+          }
+        }
+      },
+      "then": {
+        "required": ["items"]
+      },
+      "required": ["type"],
+      "dependentSchemas": {
+        "format": {
+          "oneOf": [
+            {
+              "properties": {
+                "type": {
+                  "enum": [
+                    "string"
+                  ]
+                }
+              },
+              "required": ["type"]
+            },
+            {
+              "properties": {
+                "type": {
+                  "const": "array"
+                },
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "type" : {
+                      "const" : "string"
+                    }
+                  }
+                }
+              },
+              "required": ["type"]
+            }
+          ]
+        },
```

Comment:
this is not good, format should apply on array property items with type `string`, here it applies on property of type `array`